.travis.yml

```diff
@@ -7,21 +7,23 @@ cache: yarn
 sudo: required
 services:
 - docker
-script:
-# - yarn knit -- validate --verbose
-# - yarn lint
-# - yarn flow
-- yarn test -- --no-cache
-after_success:
-- NODE_ENV=production COMMIT_HASH=$TRAVIS_COMMIT yarn build
-- docker login -e="$DOCKER_EMAIL" -u="$DOCKER_USERNAME" -p="$DOCKER_PASSWORD" quay.io
-- bash docker.sh
 before_install:
 - yarn --version
 - curl -o- -L https://yarnpkg.com/install.sh | bash -s -- --version 0.21.3
```

Comment:
travis broke our builds awhile back because they include an old version of yarn and then updated the install command to use something that was only in newer versions. 
not sure if thats still an issue, but it still has an old version of yarn